wagtail/admin/views/generic/chooser.py

```diff
@@ -208,7 +213,9 @@ def is_multiple_choice(self):
         return self.request.GET.get("multiple")
 
     @property
-    def columns(self):
+    def columns(self) -> list[Column]:
+        if self.i18n_enabled:
+            return [self.title_column, LocaleColumn()]
```

Comment:
Not sure if we've ever had ordering working in choosers before (it needs the link to be AJAX-ified). If not, then we need to disable it

```suggestion
            return [self.title_column, LocaleColumn(sort_key=None)]
```